test/Base_Tests/src/Semantic/Meta_Spec.enso

```diff
@@ -290,17 +290,21 @@ add_specs suite_builder =
                 _ : Meta.Type -> meta_typ.methods
                 _ -> Test.fail "Should be a Meta.Type: " + meta_typ.to_text
 
-            methods.sort . should_equal ['bar', 'baz', 'first_method', 'foo', 'my_method', 'other_method', 'second_method']
+            expected_methods = ['bar', 'baz', 'first_method', 'foo', 'my_method', 'other_method', 'second_method']
+            expected_methods . should_only_contain_elements_in methods
 
         group_builder.specify "static methods of MyType" <|
             methods = Meta.meta (Meta.type_of My_Type) . methods
-            methods.sort . should_equal ['Value', 'create', 'factory', 'first_method', 'my_method', 'other_method', 'second_method']
+            expected_methods = ['Value', 'create', 'factory', 'first_method', 'my_method', 'other_method', 'second_method']
+            expected_methods . should_only_contain_elements_in methods
 
         group_builder.specify "methods of Integer" <|
-            Meta.meta Integer . methods . sort . should_equal ['%', '*', '+', '-', '/', '<', '<=', '>', '>=', '^', 'abs', 'bit_and', 'bit_not', 'bit_or', 'bit_shift', 'bit_shift_l', 'bit_shift_r', 'bit_xor', 'ceil', 'div', 'fits_in_long', 'floor', 'negate', 'round', 'to_decimal', 'to_float', 'truncate']
+            expected_methods = ['%', '*', '+', '-', '/', '<', '<=', '>', '>=', '^', 'abs', 'bit_and', 'bit_not', 'bit_or', 'bit_shift', 'bit_shift_l', 'bit_shift_r', 'bit_xor', 'ceil', 'div', 'fits_in_long', 'floor', 'negate', 'round', 'to_decimal', 'to_float', 'truncate']
```

Comment:
Can you concatenate the `expected_methods` from three parts?
- those coming from `Any`
- those coming from `Number`
- those coming from `Integer`

Reply:
Done in 2edaa226ff8e3f304454c76c8dfddec1c6f15092